missing-privacy-policy.md

```diff
@@ -0,0 +1,31 @@
+# Missing Privacy Policy
+
+#### 🤔 What Happened
+
+If you're submitting your Android app to Google Play Store using the `expo upload:android` command or Google Play console, you may encouter an error saying you need to have a privacy policy set for your app.
```

Comment:
```suggestion
If you're submitting your Android app to Google Play Store using the `expo upload:android` command or Google Play console, you may encounter an error saying you need to have a privacy policy set for your app. It would look something like this on the Google Play Console:
```